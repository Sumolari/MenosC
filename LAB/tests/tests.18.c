{
	int x;
	int y;
	bool z;

    read(x);
    read(y);

    if ( true ) print( 0 );
    else {}

    if ( false ) {} else { print( 1 ); }

    if ( !true ) {} else { print( 2 ); }

    if ( !false ) { print(3); } else {}

    if ( true || false ) {
    	print( 4 );
    } else {}

    if ( true && true ) {
    	print( 5 );
    } else {}

    if ( true && !false ) {
    	print( 6 );
    } else {}

    if ( (x != y) || (x != 0) ) {
    	print( 7 );
    } else {}

    if ( !( x == 0 ) )
    	print( 8 );
    else {}

    if ( ( x == y ) && ( x > 0 ) ) {
    	print( 9 );
    }
   	else {}
}
