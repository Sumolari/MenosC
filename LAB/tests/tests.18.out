  0 EREAD             ,          ,  p:   0  
  1 EREAD             ,          ,  p:   1  
  2 EASIG     i:   1  ,          ,  p:   3  
  3 EIGUAL    p:   3  ,  i:   0  ,  e:   7  
  4 EASIG     i:   0  ,          ,  p:   4  
  5 EWRITE            ,          ,  p:   4  
  6 GOTOS             ,          ,  e:   7  
  7 EASIG     i:   0  ,          ,  p:   5  
  8 EIGUAL    p:   5  ,  i:   0  ,  e:  10  
  9 GOTOS             ,          ,  e:  12  
 10 EASIG     i:   1  ,          ,  p:   6  
 11 EWRITE            ,          ,  p:   6  
 12 EASIG     i:   1  ,          ,  p:   7  
 13 EDIF      i:   1  ,  p:   7  ,  p:   8  
 14 EIGUAL    p:   8  ,  i:   0  ,  e:  16  
 15 GOTOS             ,          ,  e:  18  
 16 EASIG     i:   2  ,          ,  p:   9  
 17 EWRITE            ,          ,  p:   9  
 18 EASIG     i:   0  ,          ,  p:  10  
 19 EDIF      i:   1  ,  p:  10  ,  p:  11  
 20 EIGUAL    p:  11  ,  i:   0  ,  e:  24  
 21 EASIG     i:   3  ,          ,  p:  12  
 22 EWRITE            ,          ,  p:  12  
 23 GOTOS             ,          ,  e:  24  
 24 EASIG     i:   1  ,          ,  p:  13  
 25 EASIG     i:   0  ,          ,  p:  14  
 26 EASIG     i:   1  ,          ,  p:  15  
 27 EIGUAL    p:  13  ,  i:   1  ,  e:  30  
 28 EIGUAL    p:  14  ,  i:   1  ,  e:  30  
 29 EASIG     i:   0  ,          ,  p:  15  
 30 EIGUAL    p:  15  ,  i:   0  ,  e:  34  
 31 EASIG     i:   4  ,          ,  p:  16  
 32 EWRITE            ,          ,  p:  16  
 33 GOTOS             ,          ,  e:  34  
 34 EASIG     i:   1  ,          ,  p:  17  
 35 EASIG     i:   1  ,          ,  p:  18  
 36 EASIG     i:   0  ,          ,  p:  19  
 37 EIGUAL    p:  17  ,  i:   0  ,  e:  40  
 38 EIGUAL    p:  18  ,  i:   0  ,  e:  40  
 39 EASIG     i:   1  ,          ,  p:  19  
 40 EIGUAL    p:  19  ,  i:   0  ,  e:  44  
 41 EASIG     i:   5  ,          ,  p:  20  
 42 EWRITE            ,          ,  p:  20  
 43 GOTOS             ,          ,  e:  44  
 44 EASIG     i:   1  ,          ,  p:  21  
 45 EASIG     i:   0  ,          ,  p:  22  
 46 EDIF      i:   1  ,  p:  22  ,  p:  23  
 47 EASIG     i:   0  ,          ,  p:  24  
 48 EIGUAL    p:  21  ,  i:   0  ,  e:  51  
 49 EIGUAL    p:  23  ,  i:   0  ,  e:  51  
 50 EASIG     i:   1  ,          ,  p:  24  
 51 EIGUAL    p:  24  ,  i:   0  ,  e:  55  
 52 EASIG     i:   6  ,          ,  p:  25  
 53 EWRITE            ,          ,  p:  25  
 54 GOTOS             ,          ,  e:  55  
 55 EASIG     p:   0  ,          ,  p:  26  
 56 EASIG     p:   1  ,          ,  p:  27  
 57 EASIG     i:   1  ,          ,  p:  28  
 58 EDIST     p:  26  ,  p:  27  ,  e:  60  
 59 EASIG     i:   0  ,          ,  p:  28  
 60 EASIG     p:   0  ,          ,  p:  29  
 61 EASIG     i:   0  ,          ,  p:  30  
 62 EASIG     i:   1  ,          ,  p:  31  
 63 EDIST     p:  29  ,  p:  30  ,  e:  65  
 64 EASIG     i:   0  ,          ,  p:  31  
 65 EASIG     i:   1  ,          ,  p:  32  
 66 EIGUAL    p:  28  ,  i:   1  ,  e:  69  
 67 EIGUAL    p:  31  ,  i:   1  ,  e:  69  
 68 EASIG     i:   0  ,          ,  p:  32  
 69 EIGUAL    p:  32  ,  i:   0  ,  e:  73  
 70 EASIG     i:   7  ,          ,  p:  33  
 71 EWRITE            ,          ,  p:  33  
 72 GOTOS             ,          ,  e:  73  
 73 EASIG     p:   0  ,          ,  p:  34  
 74 EASIG     i:   0  ,          ,  p:  35  
 75 EASIG     i:   1  ,          ,  p:  36  
 76 EIGUAL    p:  34  ,  p:  35  ,  e:  78  
 77 EASIG     i:   0  ,          ,  p:  36  
 78 EDIF      i:   1  ,  p:  36  ,  p:  37  
 79 EIGUAL    p:  37  ,  i:   0  ,  e:  83  
 80 EASIG     i:   8  ,          ,  p:  38  
 81 EWRITE            ,          ,  p:  38  
 82 GOTOS             ,          ,  e:  83  
 83 EASIG     p:   0  ,          ,  p:  39  
 84 EASIG     p:   1  ,          ,  p:  40  
 85 EASIG     i:   1  ,          ,  p:  41  
 86 EIGUAL    p:  39  ,  p:  40  ,  e:  88  
 87 EASIG     i:   0  ,          ,  p:  41  
 88 EASIG     p:   0  ,          ,  p:  42  
 89 EASIG     i:   0  ,          ,  p:  43  
 90 EASIG     i:   1  ,          ,  p:  44  
 91 EMAY      p:  42  ,  p:  43  ,  e:  93  
 92 EASIG     i:   0  ,          ,  p:  44  
 93 EASIG     i:   0  ,          ,  p:  45  
 94 EIGUAL    p:  41  ,  i:   0  ,  e:  97  
 95 EIGUAL    p:  44  ,  i:   0  ,  e:  97  
 96 EASIG     i:   1  ,          ,  p:  45  
 97 EIGUAL    p:  45  ,  i:   0  ,  e: 101  
 98 EASIG     i:   9  ,          ,  p:  46  
 99 EWRITE            ,          ,  p:  46  
100 GOTOS             ,          ,  e: 101  
101 FIN               ,          ,          
